doc/governance/accept_recovery.rst

```diff
@@ -48,6 +51,8 @@ A member proposes to recover the network and other members can vote on the propo
 
 Once the proposal to recover the network has passed under the rules of the :term:`Constitution`, the recovered service is ready for members to submit their recovery shares.
 
+Note that the ``transition_service_to_open`` proposal takes two parameters: the previous and the next service identity. This is to ensure that the correct network is recovered and to facilitate auditing, as well as to avoid forks. The previous service identity is used to validate the snapshot the recovery node is started from; CCF will refuse to start from a snapshot where the signing node certificate is not endorsed by the previous service identity. Since both identities are recorded on the ledger with the proposal, it is always clear at which point the identity changed.
```

Comment:
We should clarify that the previous identity is optional. Or more precisely, must _not_ be provided for new services, but _must_ be provided for recoveries.